components/FormAddNurse/FormAddNurse.js

```diff
@@ -33,25 +29,14 @@ export default function FormAddNurse() {
       image: getRandomImageURL(), //to generate a random image
     }; //reading the nurse data
 
-    //calling API
     try {
-      const response = await fetch(`/api/nurses`, {
-        method: "POST",
-        headers: { "Content-Type": "application/json" }, // Set the content type to JSON
-        body: JSON.stringify(nurseData), //Adding in body the new nurse
-      });
-
-      if (!response.ok) {
-        const responseData = await response.json();
-        console.error("Error adding nurse:", responseData.message);
-        return;
-      }
+      await onSubmitNurse(nurseData);
       mutate();
       setSelectedImage(getRandomImageURL());
```

Comment:
You're getting an error because `mutate` and `setSelectedImage` don't exist here anymore.

Reply:
Done! Thanks a lot!!!!